lib/manageiq/appliance_console/cli.rb

```diff
@@ -232,6 +249,23 @@ def set_external_db
       config.post_activation
     end
 
+    def replication
+      if options[:replication] == "primary"
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationPrimary.new
+        say("Configuring Server as Primary")
+      else
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationStandby.new
+        say("Configuring Server as Standby")
+        db_replication.disk = disk_from_string(options[:dbdisk])
+        db_replication.primary_host = options[:primary_host]
+        db_replication.standby_host = options[:standby_host]
+        db_replication.run_repmgrd_configuration = options[:repmgrd] ? "enabled" : "disabled"
+      end
+      db_replication.node_number = options[:cluster_node_number]
+      db_replication.database_password = options[:password]
```

Comment:
You also need to set `database_name` and `database_user` if they were provided. Both the classes set sane defaults (vmdb_production/root), but we should be able to change those using the cli.